tests/pipelines/test_pipelines_image_to_text.py

```diff
@@ -143,6 +145,56 @@ def test_large_model_pt(self):
             ],
         )
 
+    @slow
+    @require_torch
+    def test_generation_blip(self):
```

Comment:
Can we add `pipeline(..., prompt="some prompt")` in a fast test ? So we make sure we don't break it ?